plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginService.java

```diff
@@ -43,6 +43,9 @@
  */
 public class GoogleLoginService implements IGoogleLoginService {
 
+  private static final String OAUTH_DATA_STORE_PREFERENCE_PATH =
```

Comment:
This is the limitation I mentioned yesterday: we're persisting a single account.  How do you envision this working with multiple accounts?  (And I'm totally fine with punting on this for now.)

Hmm, I suppose `ide-login` could use:
```
Preferences prefs = Preferences.userRoot().node(preferencePath).node(credential.getStoredEmail());
```
as the node, and then have a single key "active" -> email to note the active account.  (_active_ is ok since it's not a valid email address.)